coalib/bearlib/abstractions/Linter.py

```diff
@@ -777,7 +777,7 @@ def linter(executable: str,
     ... class YLintBear:
     ...     @staticmethod
     ...     def create_arguments(config_file):
-    ...         return "--lint", filename
+    ...         return "--lint", self.file_dict.keys()
```

Comment:
you have to remove `@staticmethod` and add `self` in the function signatur, otherwise `self` is not available here ;)